### PR TITLE
Hw7 extra task

### PR DESCRIPTION
**Pull request by screamofthepast**

### CODE REVIEW

File: hw7.log [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+Voronezh
+7
+
+[root@localhost home]# nano script.sh
+#!/bin/bash
+trusted=(root)
+
+allusers=$(users)
+date=$(date)
+if [[ "${trusted}" = "${allusers}" ]]; then
```
Comment (Cuttlerat) [inline hw7.log]: == instead of =

File: hw7.log [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+Rostov-on-Don
+Perm
+Voronezh
+7
+
+[root@localhost home]# nano script.sh
+#!/bin/bash
+trusted=(root)
+
+allusers=$(users)
```
Comment (Cuttlerat) [inline hw7.log]: Would be better to define it as array

File: hw7.log [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+date=$(date)
+if [[ "${trusted}" = "${allusers}" ]]; then
+        exit 0
+else
+        for log in ${allusers}
```
Comment (Cuttlerat) [inline hw7.log]: Please use array

File: hw7.log [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+        do
+                if [[ "${trusted}" != "${log}" ]]; then
+                        hackers+=($log)
+                fi
+        done
+        for user in ${hackers}
```
Comment (Cuttlerat) [inline hw7.log]: Same

File: hw7.log (hunk 2 above)
Comment (screamofthepast) [reply hw7.log]: when i try to define as array bash write this

> ./script.sh: line 4: allusers[@]: bad array subscript

Comment (screamofthepast) [reply hw7.log]: or you mean this?
`allusers+=$(users)`

**Comment by screamofthepast** [comment]
fixed all comments

**Pull request closed without merging**